src/main/site/site.xml

```diff
@@ -77,12 +77,17 @@
       <item name="Bulk Loads" href="book.html#arch.bulk.load" target="_blank" />
       <item name="Metrics" href="metrics.html" target="_blank" />
       <item name="HBase on Windows" href="cygwin.html" target="_blank" />
-      <item name="Cluster replication" href="replication.html" target="_blank" />
-    </menu>
-    <menu name="0.94 Documentation">
-      <item name="API" href="0.94/apidocs/index.html" target="_blank" />
-      <item name="X-Ref" href="0.94/xref/index.html" target="_blank" />
-      <item name="Ref Guide (single-page)" href="0.94/book.html" target="_blank" />
+      <item name="Cluster replication" href="book.html#replication" target="_blank" />
+      <item name="1.2 Documentation">
+        <item name="API" href="1.2/apidocs/index.html" target="_blank" />
+        <item name="X-Ref" href="1.2/xref/index.html" target="_blank" />
+        <item name="Ref Guide (single-page)" href="1.2/book.html" target="_blank" />
+      </item>
+      <item name="1.1 Documentation">
```

Comment:
Do we need 1.1 Documentation?

Reply:
Good question. This menu item came in as a part of one of the version change updates. We still host the 1.1 javadocs and ref guide, so there's no harm pointing to them. There is a later commit that removes 1.1 (as evidenced by us no longer linking to 1.1 on the main website). I could track that down and include it as well if you like.

Reply:
Since 1.1 docs are still there we can add it. I haven't checked it before.